airflow/models/baseoperator.py

```diff
@@ -243,6 +243,8 @@ class derived from this one results in the creation of a task object,
     :param do_xcom_push: if True, an XCom is pushed containing the Operator's
         result
     :type do_xcom_push: bool
+    :param task_tags: List of tags used to identify this task
```

Comment:
```suggestion
    :param tags: List of tags used to identify this task
```

We don't need to say "task" here -- it's already on a task.

Reply:
It was originally just called `tags` but when some tests failed I found out that [ECSOperator](https://github.com/apache/airflow/blob/e2b219823b9435dcdb0956a64ee2fdbf2e475124/airflow/providers/amazon/aws/operators/ecs.py#L89-L90), [OpsgenieAlertOperator](https://github.com/apache/airflow/blob/e2b219823b9435dcdb0956a64ee2fdbf2e475124/airflow/providers/opsgenie/operators/opsgenie_alert.py#L50-L51) and [AzureContainerInstancesOperator](https://github.com/apache/airflow/blob/e2b219823b9435dcdb0956a64ee2fdbf2e475124/airflow/providers/microsoft/azure/operators/azure_container_instances.py#L90-L91) already have a `tags` parameter.